### DISCUSSION TITLE
Workflow Engine going to sleep mode after inactive some time

### DISCUSSION THREAD

**Discussion by kpkrmca** [optimajet/WorkflowEngine.NET#213] [General]
Hi Team,

We have one of the .Net Core API application that uses workflow engine for automating the API Calls ,but over the weekend its going to sleep mode / inactive state below is the error message for referrence and Next Timer is not being called to proceed next activities.

http error 500.37 - ANCM failed to start within startup time limit.

how ever we verified the IIS setting its Always running and idle time out is 0.

Is there any configuration from code/conf level which we can update and it will not go to sleep/inactive mode.

**Comment by optimajet** [comment]
@kpkrmca 

> how ever we verified the IIS setting its Always running and idle time out is 0.

There's something wrong with the application or IIS settings. There is no sleep functionality in the Workflow Engine library itself.
